src/lib/utils/time.ts

```diff
@@ -36,7 +36,8 @@ export const timeSince = (date: Date) => {
   }
 }
 
-export const categorizeDate = (date: Date): string | number => {
+//returrn an objecct with  { key: "today", label: "Today", isOpen: true }
```

Comment:
Looks like a perfect spot to use an actual type?

Reply:
Yes and also this shouldn't be in this pr.. Sorry for that there's something i'm missing in git